hack/jenkins/e2e-runner.sh

```diff
@@ -257,12 +257,19 @@ case "${KUBERNETES_PROVIDER}" in
             cp /var/lib/jenkins/gce_keys/google_compute_engine ${WORKSPACE}/.ssh/
             cp /var/lib/jenkins/gce_keys/google_compute_engine.pub ${WORKSPACE}/.ssh/
         fi
-        if [[ ! -f ${WORKSPACE}/.ssh/google_compute_engine ]]; then
-            echo "google_compute_engine ssh key missing!"
+        echo 'Checking existence of private ssh key'
+        local _key="${WORKSPACE}/.ssh/google_compute_engine"
+        if [[ ! -f "${_key}" || ! -f "${_key}.pub" ]]; then
+            echo 'google_compute_engine ssh key missing!'
             exit 1
         fi
+        echo "Checking presence of public key in ${PROJECT}"
+        if ! gcloud compute "--project=${PROJECT}" project-info describe |
```

Comment:
I'm wary of `gcloud compute` piped into `grep`. Is it possible to use a format flag to ensure that we get exactly what we want, even if the default format changes?